### PR TITLE
⬆️ Gatsby V2

### PR DESCRIPTION
**Pull request by donofkarma**
Making some changes to align with the official release of Gatsby V2.

### CODE REVIEW

**Comment by peduarte** [comment]
Deploy preview for *fathomlondon* ready!

Built with commit c3b99a40419ceb6334654143c8de88d1bb9dac46

https://deploy-preview-334--fathomlondon.netlify.com

File: .editorconfig [hunk 1]
```diff
@@ -4,6 +4,7 @@ root = true
 [*]
 charset = utf-8
 indent_style = tab
+indent_size = 2
```
Comment (petecorreia) [inline .editorconfig]: why was this added?

File: src/layouts/index.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ class Layout extends React.Component {
 		return (
 			<StaticQuery
 				query={graphql`
-					query LayoutQuery {
+					{
```
Comment (petecorreia) [inline src/layouts/index.js]: curious why this was removed?

Comment (donofkarma) [reply src/layouts/index.js]: Query names are no longer required and seeing as the graphql query is directly tied to the render in  `<StaticQuery />` we don't need a name to help with readability as the connection is already explicit.

https://www.gatsbyjs.org/docs/migrating-from-v1-to-v2/#explicit-query-names-no-longer-required

File: .editorconfig (hunk 1 above)
Comment (donofkarma) [reply .editorconfig]: This was specified on other projects so I thought I'd add for consistency. Can remove if required.

Comment (petecorreia) [reply .editorconfig]: I don’t see it in bp, capptivate and vfunction (recent projects). would rather this stay aligned

File: src/layouts/index.js (hunk 2 above)
Comment (petecorreia) [reply src/layouts/index.js]: makes sense 👍

File: .editorconfig (hunk 1 above)
Comment (donofkarma) [reply .editorconfig]: Ok, I'll pull it out.

**Comment by AndrewJDick** [comment]
@petecorreia can you re-approve please? ^
